### DISCUSSION TITLE
Is there a high-res/vector Sming-logo?

### DISCUSSION THREAD

**Discussion by callimero** [SmingHub/Sming#2821] [General]
As the topic says. Would love to use it in my article, but the png is soooo small.

And: Is the left and right border cropped intentionally?

Searched at google and in github...

![00_sming-logo](https://github.com/SmingHub/Sming/assets/7713008/f955e43c-2f5b-4332-95db-d62d94b82d80)

Cheers,
Carsten

**Comment by slaff** [comment]
Carsten, take a look in this directory https://github.com/SmingHub/Sming/tree/gh-pages/images .
The files that might interest you are sming-logo-hires.png and **sming.ai**. The second one is an Adobe file that might work for you.

All credits for the logo go to https://github.com/DominicMe.

See here for details: https://github.com/SmingHub/Sming/issues/829

Hope this helps.

**Reply by callimero** [reply]
Thanks! Why did I miss that... 

Sorry, thanks for you patience with me :)
